docs/pages/demo/datetime-picker/index.mdx

```diff
@@ -46,4 +47,4 @@ Here are some examples of heavily customized date & time pickers
 
 #### Components API
 
-<LinkedComponents components={['DateTimePicker', 'KeyboardDateTimePicker']} />
+<LinkedComponents components={['DateTimePicker', 'MobileDateTimePicker']} />
```

Comment:
3 others pages of the documentation are impacted. I think that whatever solution we go with, it should impact them too.